src/routes/cache-hooks/cache-hooks.service.ts

```diff
@@ -78,6 +78,7 @@ export class CacheHooksService {
       // - the list of all executed transactions for the safe
       // - the list of module transactions for the safe
       // - the safe configuration
+      // - the list of Safes for the module
```

Comment:
Adding/Removing a module is not necessarily a module transaction I believe (from the Safe Transaction Service).

Reply:
That's a good point and something I've also thought about, it doesn't imply a module transaction, in fact, I think it should be a Multisig one, right? In that case, we would need to clear also on `EXECUTED_MULTISIG_TRANSACTION` event.

Reply:
A Module should be able to technically change the state of the Safe so I believe that having this under a Module Event is correct but not complete. An `EXECUTED_MULTISIG_TRANSACTION` should also be able to change the state of the Safe.

Reply:
Actually it is likely that both of these events can have the same side-effects. We should consider merging them if that's the case.

Reply:
After discussion and due to the required changes for clearing the cache on `EXECUTED_MULTISIG_TRANSACTION` events, I have added a `TODO` in the relevant place and will tackle this in a follow up PR.

Reply:
As our UI deploys a new recovery module per Safe we can also assume that the Safes-module relationship won't change so invalidating the cache shouldn't be an issue for now. (I've updated the aforementioned comment accordingly.)